third_party/gloo-latest/gloo.yaml

```diff
@@ -365,12 +365,6 @@ rules:
 - apiGroups: ["gloo.solo.io"]
   resources: ["settings", "upstreams","upstreamgroups", "proxies","virtualservices"]
   verbs: ["*"]
-- apiGroups: ["networking.internal.knative.dev"]
```

Comment:
This should be done separately.  cc @ilackarms

Reply:
Should I back out this?

Reply:
Yes, please separate that into a different PR.  Thanks.

Now that we have Gloo tests we have more ways to testing ClusterIngress/Ingress related changes by running `/test pull-knative-serving-gloo-0.17.1`.  That will probably be useful for this separated out PR as well as #5689

Reply:
It's been reverted.   https://github.com/knative/serving/issues/5708 has bee filed.